Magi/Milia of the North Coven/Milia, the Animal Speaker.md

```diff
@@ -41,35 +37,33 @@ Under the teachings of the witch, Milia refined her ability to summon animals an
 **Soak:** +1  
 **Fatigue Leves:** OK, 0, -1, -3, -5, Unconscious  
 **Wound Penalties:** -1 (1-5), -3 (6-10), -5 (11-15), Incapacitated (16-20), Dead (21+)  
-  
 ## Abilities  
-  
 + Animal Handling 2 (birds)  
 + Animal Ken 4 (birds)  
 + Area Lore: Mirna River 2 (birds)  
 + Athletics 1 (running)  
 + Awareness 2 (alertness)  
 + Brawl 1 (dodge)
 + Craft: Wood 2 (musical instruments)  
++ Cursing 2 (disease)  
 + Dowsing 2 (vis)  
 + Flight 2 (speed)  
 + Folk Ken 1 (witches)  
 + Folk Witch Magic Theory 3 (Healing)  
-+ Folk Witches Lore 2 (rituals)  
 + Guile 1 (peasants)  
 + Healing 2 (wounds)  
 + Herbalism 3 (theriacs)  
-+ Hex 2 (face to face)  
 + Hunt 1 (traces)  
--  Language (native): South Slavonic 5 (Serbian)
-+ Magic Lore 2 (magical animals)  
+-  Language (native): South Slavonic 5 (Serbian)  
```

Comment:
Shouldn't this line start with `+` instead of `-` ?

Reply:
Probably, the rendered markdown it's the same but at least it'll be uniform.

Reply:
It looks like you changed the line below (making it two lines in a row with a mismatched bullet point).

Reply:
> It looks like you changed the line below (making it two lines in a row with a mismatched bullet point).

I should have corrected the error, there might other lines like these in other character's sheets however.